test_container/tests/test/standard-flavor/all/import_modules.py

```diff
@@ -195,19 +224,23 @@ def run(ctx):
                     module_limit = module_limit,                                                                     
                     excluded_modules = excluded_modules,
                     excluded_submodules = excluded_submodules)
-                importer.import_modules()
+                importer.import_modules(ctx.root_module_name)
             /
             '''))
-        #with UdfDebugger(test_case=self):
-        rows = self.query('''SELECT import_all_modules.import_all_modules() FROM dual''')
-        print("Number of modules:",len(rows))
-        failed_imports = [(row[0],row[1]) for row in rows if row[2] == "ERROR"]
-        for i in failed_imports:
-            print(i[0])
-        for i in failed_imports:
-            print(i[0], i[1])
-        self.assertEqual(failed_imports,[])
 
+    def test_import_all_modules(self):
```

Comment:
Do you know what this costs us regarding test time. I assume the tests take several minutes longer now. However, we probably need to live with it.

Reply:
When I run the test locally on my machine, it takes 1m7,650s. I think that's acceptable.